lib/browser/guest-view-manager.js

```diff
@@ -184,6 +184,32 @@ const attachGuest = function (embedder, elementInstanceId, guestInstanceId, para
     disableBlinkFeatures: params.disableblinkfeatures
   }
 
+  // parse the 'webpreferences' attribute string, if set
+  // this uses the same parsing rules as window.open uses for its features
```

Comment:
@pfrazee The issue was that new JS files need to be added to `filenames.gypi` to be included in the built version.

I pushed a couple changes to this pull request, please let me know what you think.